### PR TITLE
Add matomo tracking to genetics app

### PR DESCRIPTION
**Pull request by alepev**
Relates to https://github.com/thehyve/opentargets-genetics-automation/pull/34

[Original issue description](https://trello.com/c/dm9LqvEn/219-make-matomo-a-configurable-option-in-otg-automation)

### CODE REVIEW

File: apps/genetics/public/matomo/init_matomo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  var pushState = history.pushState;
+  history.pushState = function() {
+    pushState.apply(history, arguments);
+    registerUserVisit();
+  };
+  registerUserVisit();
+})();
+
+(function() {
+  var u = 'https://localhost:9443/';
```
Comment (RobinM-code) [inline apps/genetics/public/matomo/init_matomo.js:26]: Can you make this url configurable in the `.env`? Ofc we are not always deploying on localhost

**Review by RobinM-code — CHANGES_REQUESTED** [review]

File: apps/genetics/public/matomo/init_matomo.js (hunk 1 above)
Comment (fedde-s) [reply apps/genetics/public/matomo/init_matomo.js:26]: It should be feasible by either inserting the URL into `apps/genetics/public/config.js` (like other settings) and reading from the global `window` object here, or, if this runs _before_ `config.js`, leaving a placeholder word to insert it right here, as this code file is in `public/` and served to the browser un-transpiled.

**Comment by alepev** [comment]
@RobinM-code (and others) see changes in last commit + related commit in [otg-automation](https://github.com/thehyve/opentargets-genetics-automation/pull/34/commits/37fc5c8f629a847ca1edbe808f53928f8453c96b). this doesn't seem to work, i.e. I can install Matomo but it doesn't seem to track user activity

**Comment by fedde-s** [comment]
> see [changes in last commit](https://github.com/thehyve/ot-ui-apps/pull/4/commits/16a2290cd5f935b7fd299b6fea01db360306ff8e) + [related commit in otg-automation](https://github.com/thehyve/opentargets-genetics-automation/pull/34/commits/37fc5c8f629a847ca1edbe808f53928f8453c96b). this doesn't seem to work

We've since solved it in a different way, but for context on why I expect it didn't work:  
`import` statements in the files in `src/` are parsed during transpilation/bundling. `matomo_init.js` Is in `public/` and kept as an unprocessed, statically served file. By the time the browser runs it, the `src/` folder does not exist, as the contents of `src/` are minified into the bundles.

File: apps/genetics/public/matomo/init_matomo.js (hunk 1 above)
Comment (fedde-s) [reply apps/genetics/public/matomo/init_matomo.js]: Should be done now.

File: apps/genetics/public/matomo/init_matomo.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  var pushState = history.pushState;
+  history.pushState = function() {
+    pushState.apply(history, arguments);
+    registerUserVisit();
+  };
+  registerUserVisit();
+})();
+
+(function() {
+  var u = 'DISABLED';
```
Comment (RobinM-code) [inline apps/genetics/public/matomo/init_matomo.js:26]: But now it's hard coded to be 'DISABLED' right?

**Comment by alepev** [comment]
Ready for final review

**Review by RobinM-code — APPROVED** [review]
Perfecto!

**Pull request merged**